### DISCUSSION TITLE
Automation turning off

### DISCUSSION THREAD

**Discussion by charakaSamawry** [darkreader/darkreader#7248] [Q&A] [answered]
Whenever i manually (or using hotkey/keyboard shortcut) disable/enable the dark mode i have to go and recheck the automation in the settings gui. Cant this stay checked and revert to automatic after. Sometimes a site is broken or doesnt work with darkreader enabled and site specific enabling disabling also doesnt fix it and then afterwards it would be useful if dark reader reverted to using automation, either the next day or night cycle, ie. it automatically switches on the next automatic cycle but is already in dark/light mode, then on the next automatic cycle it then switches to dark/light 
(confusing i know lol but say its night, and i switch to day/light mode, then the next morning it would switch to day/light as normal, but it is already in day/light mode so nothing changes, but then the next night it should still revert to the automation and switch to night/dark mode)

**Comment by Gusted** [comment, marked as answer]
See #3268

**Reply by charakaSamawry** [reply]
Thanks!
